pepper-apis/dss-core/src/test/java/org/broadinstitute/ddp/route/UserRegistrationRouteStandaloneTest.java

```diff
@@ -419,7 +420,20 @@ public void testRegister_newUser() {
             // Have to create a completely new user that only exists in Auth0, so that email-lookup-by-user will not
             // fail and we can test that user/profile is created post-registration.
             var mgmtClient = Auth0Util.getManagementClientForDomain(handle, auth0Domain);
-            return Auth0Util.createTestingUser(mgmtClient).getAuth0Id();
+            Auth0Util.TestingUser testAuth0User = Auth0Util.createTestingUser(mgmtClient);
+            assertNotNull(testAuth0User.getEmail());
+            try {
+                //delay to make sure no timings issue
+                Thread.sleep(5 * 1000);
```

Comment:
Test seem to be failing intermittently and looks like timing issue. Added 5sec delay which seem to help